README.md

```diff
@@ -2,7 +2,22 @@
 
 This repository is a monorepo for smart-contract development in AssemblyScript for Massa blockchain.
 
-## Build
+## Packages:
+
+### @massalabs/sc-project-initializer
+
+This package makes your first steps into massa smart-contracts development easy:
+
+- [Installation](./packages/sc-project-initializer/README.md)
+- [Build your first smart-contract](./packages/sc-project-initializer/commands/init/README.md)
+
+### @massalabs/sc-deployer
+
+This package is a tool to deploy your smart-contracts on the Massa network:
```

Comment:
Sc-deployer is a tool used for deploying your smart contracts on the Massa network: